### PR TITLE
Nqt+1 template mail

### PR DESCRIPTION
**Pull request by ethanmills**
Replace the blank template with a standard template

### CODE REVIEW

**Review by xswoodsx — APPROVED** [review]
LGTM, thanks very much for the changes.

File: app/mailers/school_mailer.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ class SchoolMailer < ApplicationMailer
   BASIC_TEMPLATE = "b1ab542e-a8d5-4fdf-a7aa-f0ce49b98262"
```
Comment (FriedSock) [inline app/mailers/school_mailer.rb:19]: I think we can remove this constant now as well since nothing will be using it

**Review by FriedSock — APPROVED** [review]
Looks good if this does work with notify

File: app/mailers/school_mailer.rb (hunk 1 above)
Comment (ethanmills) [inline app/mailers/school_mailer.rb:19]:
```suggestion
```

**Pull request merged**